cmd/ztest.c

```diff
@@ -3624,6 +3732,14 @@ ztest_vdev_attach_detach(ztest_ds_t *zd, uint64_t id)
 		goto out;
 	}
 
+	/*
+	 * Does not work with expandable raidz, bp corruptions detected.
```

Comment:
There is an unresolved issue with ztest_vdev_attach_detach() and raidz expansion.
It was mentioned here: https://github.com/ahrens/zfs/pull/37.
It is not clear for me, why it happen, but it could be seen, that it happen when some of raidz children become a mirrors.

Reply:
Would be nice to root cause this but it is not critical for the first release .  An in-progress expansion will pause when a replacing/resilvering is taking place which is the only practical way for a raidz leaf to become a mirror.

I can update the comment to be more descriptive of the reason we opt-out here during raidz expansion.